src/Features/CSharp/Portable/UseExpressionBody/Helpers/UseExpressionBodyForIndexersHelper.cs

```diff
@@ -52,10 +51,9 @@ protected override IndexerDeclarationSyntax WithBody(IndexerDeclarationSyntax de
             throw new InvalidOperationException();
         }
 
-        protected override IndexerDeclarationSyntax WithGenerateBody(
-            SemanticModel semanticModel, IndexerDeclarationSyntax declaration, OptionSet options, ParseOptions parseOptions)
+        protected override IndexerDeclarationSyntax WithGenerateBody(SemanticModel semanticModel, IndexerDeclarationSyntax declaration)
         {
-            return WithAccessorList(semanticModel, declaration, options, parseOptions);
+            return WithAccessorList(semanticModel, declaration);
```

Comment:
nit: consider `=>`

Reply:
I like the empty (brace) line separating the noise above and below. Okay to keep it this way or is it inconsistent?

Reply:
it's a nit.  so that always means (to me): up to you.